core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/common/DetailedStackStatus.java

```diff
@@ -52,6 +52,14 @@
     DOWNSCALE_IN_PROGRESS(Status.UPDATE_IN_PROGRESS),
     DOWNSCALE_COMPLETED(Status.AVAILABLE),
     DOWNSCALE_FAILED(Status.AVAILABLE),
+    // Upscale by starting statuses
+    UPSCALE_BY_START_REQUESTED(Status.UPDATE_REQUESTED),
+    UPSCALE_BY_START_IN_PROGRESS(Status.UPDATE_IN_PROGRESS),
+    UPSCALE_BY_START_FAILED(Status.AVAILABLE),
+    // Downscale by stopping statuses
+    DOWNSCALE_BY_STOP_REQUESTED(Status.UPDATE_REQUESTED),
+    DOWNSCALE_BY_STOP_IN_PROGRESS(Status.UPDATE_IN_PROGRESS),
+    DOWNSCALE_BY_STOP_FAILED(Status.AVAILABLE),
```

Comment:
Will we be able to query these FAILURE states ... DOWNSCALE_BY_STOP_FAILED / UPSCALE_BY_START_FAILED - if additional decisions need to be taken in Periscope based on these states?

Reply:
We do have stack and cluster object in Periscope, so yes we can query. We can use these states to send data to EDH and query there as well.